dynamodb/src/main/scala/akka/stream/alpakka/dynamodb/DynamoSettings.scala

```diff
@@ -52,11 +52,13 @@ final class DynamoSettings private (
 
 object DynamoSettings {
 
+  val ConfigPath = "akka.stream.alpakka.dynamodb"
```

Comment:
Because it is a [constant](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods). I named the [config path in S3 connector also the same](https://github.com/akka/alpakka/blob/6c228ba6b4e4514f97610925efd86ead7a3aa0e2/s3/src/main/scala/akka/stream/alpakka/s3/settings.scala#L180).